src/palindrom.js

```diff
@@ -11,9 +11,20 @@ if(typeof require !== 'undefined') {
   var JSONPatchOT = require('json-patch-ot');
   var JSONPatchOTAgent = require('json-patch-ot-agent');
   var URL = require('./URL');
+  var axios = require('axios');
+
+  var NodeWebSocket = require('websocket').w3cwebsocket;
+
+  /* this allows us to stub WebSockets */
+  if(!global.WebSocket) { /* we are in production env */
+    var WebSocket = NodeWebSocket;
```

Comment:
@warpech I guess after Webpack it's the same. Would you like me to write a sample?

And if you think about it, since this `if` is executed at runtime. The required module should be available for importing all the time in case the condition was `false`, right?. Requiring it inside an if doesn't change anything bundle-size-wise.